### PR TITLE
API: added the API reference decorator

### PR DESCRIPTION
**Pull request by irfanpena**
Added the API Reference decorator in the DTOs and Controller files.
- Assistant
- Chat
- Cortex
- Model
- Messages
- Threads

### CODE REVIEW

**Comment by louis-jan** [comment]
Please help resolve again @irfanpena

**Comment by irfanpena** [comment]
Done @louis-jan , kindly check again

File: cortex-js/src/infrastructure/dtos/models/model-successfully-created.dto.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    @ApiProperty({ type: [String], example: ['7B', 'Merged', 'Featured'] })
+    tags: string[];
+
+    @ApiProperty({ example: 4370000000 })
+    size: number;
+
+    @ApiProperty({ example: 'https://raw.githubusercontent.com/janhq/jan/main/models/trinity-v1.2-7b/cover.png' })
+    cover: string;
+
+    @ApiProperty({ example: 'nitro' })
```
Comment (louis-jan) [inline cortex-js/src/infrastructure/dtos/models/model-successfully-created.dto.ts:64]:
```suggestion
    @ApiProperty({ example: 'cortex' })
```

**Review by louis-jan — APPROVED** [review]
LGTM

**Comment by irfanpena** [comment]
Can I merge this? @louis-jan

**Comment by louis-jan** [comment]
> Can I merge this? @louis-jan

Yes please @irfanpena, resolve conflict and proceed.

**Pull request merged**